tests/system/HTTP/CURLRequestDoNotShareOptionsTest.php

```diff
@@ -559,6 +559,20 @@ public function testSSLWithBadKey()
         ]);
     }
 
+    public function testProxyuOption()
+    {
+        $this->request->request('get', 'http://example.com', [
+            'proxy' => 'http://localhost:3128',
```

Comment:
I think `proxy` should be an array that can set `CURLOPT_PROXY`, `CURLOPT_PROXYTYPE `, `CURLOPT_PROXYPORT`  and etc.

Reply:
It seems Guzzle does not have such options.
https://docs.guzzlephp.org/en/stable/request-options.html#proxy